bin/libs/component.ts

```diff
@@ -418,6 +418,22 @@ export function processComponentApimlStaticDefinitions(componentDir: string): bo
 
         const contents = xplatform.loadFileUTF8(path,xplatform.AUTO_DETECT);
         if (contents) {
+          const zosmfScheme = std.getenv("ZOSMF_SCHEME");
+          const attls = std.getenv("ZWE_zowe_network_client_tls_attls"); 
```

Comment:
This will not work for configs where people have recommended enabling attls through only the server variable.

It would be better to say if the client variable isn't defined, to use the server variable's value.
Something like we do here:

https://github.com/zowe/zlux-server-framework/blob/v3.x/staging/lib/util.js#L510-L520